package/libs/gettext-full/Makefile

```diff
@@ -23,7 +23,6 @@ PKG_CPE_ID:=cpe:/a:gnu:gettext
 
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=0
```

Comment:
So this is a package that is explicitly setting it to 0.  I'm all for removing the =1 where that matches the new default, but I think changing the setting of a package that has already got to the explicit trouble of disabling parallel builds should be left alone _for this commit_ at least.  By all means separately as a follow up see if it still has problems, but it seems hazardous to blend it into this large commit

Reply:
Can't reproduce any build issues